crates/bevy_render/src/texture/mod.rs

```diff
@@ -107,11 +107,16 @@ impl Plugin for ImagePlugin {
             .world()
             .get_resource::<bevy_asset::processor::AssetProcessor>()
         {
-            processor.register_processor::<bevy_asset::processor::LoadAndSave<ImageLoader, CompressedImageSaver>>(
-                CompressedImageSaver.into(),
-            );
-            processor
-                .set_default_processor::<bevy_asset::processor::LoadAndSave<ImageLoader, CompressedImageSaver>>("png");
+            processor.register_processor::<bevy_asset::processor::LoadTransformAndSave<
+                ImageLoader,
+                bevy_asset::transformer::IdentityAssetTransformer<Image>,
+                CompressedImageSaver,
```

Comment:
Bleh, not my favorite. But CompressedImageSaver needs rewriting anyways: https://github.com/bevyengine/bevy/issues/14671#issuecomment-2307855809, so it's fine.

Reply:
Yeah it's definitely ugly. I imagine you'd want to separate out the compression stage from `CompressedImageSaver` into an `AssetTransformer`, but I definitely consider it out of the scope of this work.